src/Ema/Server.hs

```diff
@@ -354,6 +354,9 @@ wsClientShim =
             } else {
               console.log("ema: ✍ Patching DOM");
               setHtml(document.documentElement, evt.data);
+              if ("emaHotReloadHook" in window) {
+                emaHotReloadHook();
```

Comment:
You want a `console.log` here, so that we can use browser console log for latter debugging at runtime.